media/transcoder/api/Transcoder.Samples.Tests/CreateJobFromAdHocTests.cs

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System.Threading;
+using Xunit;
+using Google.Cloud.Video.Transcoder.V1;
+
+namespace Transcoder.Samples.Tests
+{
+    [Collection(nameof(TranscoderFixture))]
+    public class CreateJobFromAdHocTest
+    {
+        private TranscoderFixture _fixture;
+        private readonly CreateJobFromAdHocSample _createSample;
+        private string _jobId;
+        private readonly GetJobStateSample _getSample;
+        private readonly DeleteJobSample _deleteSample;
+
+        public CreateJobFromAdHocTest(TranscoderFixture fixture)
+        {
+            _fixture = fixture;
+            _createSample = new CreateJobFromAdHocSample();
+            _getSample = new GetJobStateSample();
+            _deleteSample = new DeleteJobSample();
+        }
+
+        [Fact]
+        public void CreatesJobFromAdHoc()
+        {
+            string outputUri = "gs://" + _fixture.BucketName + "/test-output-adhoc/";
+            // Run the sample code.
+            var result = _fixture.TranscoderChangesPropagated.Eventually(() => _createSample.CreateJobFromAdHoc(
+                projectId: _fixture.ProjectId, location: _fixture.Location,
+                inputUri: _fixture.InputUri, outputUri: outputUri));
+
+            Assert.Equal(result.JobName.LocationId, _fixture.Location);
```

Comment:
The first parameter on Assert.Equal is the expected value, the second the actual value, please change this and all others around.
```suggestion
            Assert.Equal(_fixture.Location, result.JobName.LocationId);
```

Reply:
removed